docs/tutorials/tutorial-a-guide-to-using-the-snowplow-inspector-extension/index.md

```diff
@@ -0,0 +1,173 @@
+Each type of registry may require additional configuration like API keys to function.
+
+Once configured, schemas from the registry should load in the ‘Manage Schemas’ view, and events using those schemas will be checked for validation.
+
+Now that the extension can query your private Iglu repositories the extension will be able to tell you in real time whether your events and entities are failing validation from entirely within the browser.
+An example of an event passing validation is shown below:
+
+![A Self Describing Event called 'example_event' is displayed in the Snowplow Inspector extension. It contains a property 'example_field_1' with the value 'abc'. The extension says the event is Valid against its schema, signalled by green text.](images/event-passing-validation.png)
+
+An example of an event failing validation is shown below, the reason for this event failing validation is due to “example_field_1” being specified as allowing only the "string" type, meaning it cannot be null.
+Hovering over the "Invalid" warning displays text describing which aspect of the payload failed the validation.
+Clicking the warning copies this error to your clipboard.
+
+![A Self Describing Event called 'example_event' is displayed in the Snowplow Inspector extension. It contains a property 'example_field_1' with a null value. The extension says the event is Invalid against its schema, signalled by a bright red background.](images/event-validation-failed.png)
+
+This is useful, but when it comes to the debugging of failed events and the investigation of exactly WHY a particular event has failed validation there are additional tools that should be considered.
+
+#### Local registries
+Usually used for testing changes of schemas still in development, or quickly including a schema when you don't have the full access required to configure a registry.
+These schemas are managed within the extension and stored on your own machine.
+
+The only required configuration is a name, and the schemas themselves.
+
+#### Data Structures API registries
+This type is recommended for use with [Snowplow BDP](/docs/getting-started-with-snowplow-bdp/index.md) using the [Data Structures API](/docs/understanding-tracking-design/managing-data-structures-via-the-api-2/index.md).
+
+In order to function, the extension requires:
+
+- Organization ID: This is usually found in the URL when logged into the Snowplow BDP console. See more at [Managing Console API authentication](/docs/using-the-snowplow-console/managing-console-api-authentication/index.md#credentials-ui-v2).
+- API Key: When logged into the Snowplow BDP console, should be available in [API keys for managing Snowplow](https://console.snowplowanalytics.com/credentials) (within "Manage organization"). See more at [Managing Console API authentication](/docs/using-the-snowplow-console/managing-console-api-authentication/index.md#credentials-ui-v2).
+
+Optionally you may specify:
```

Comment:
This left me scratching my head, do we know that there is any alternative implementation?

Reply:
I'm now thinking this is probably there to test against Next? However this could be confusion to customers and is really only relevant to Snowplow engineers, I would guess

Reply:
It's mostly here for completion, do you think it would be better off undocumented or with those use-cases explained?

Reply:
I'm inclined to suggest that we remove it, if I were a customer most likely after reading this I'd reach out to support and ask what are the alternative DS API implementations :)